addon/lib/document.js

```diff
@@ -86,25 +86,57 @@ export default EmberObject.extend({
     const segments = slugWithPath.split(".");
     const slug = segments.pop();
     const doc = this.resolveDocument(segments);
-    return doc && doc.fields.find(field => field.question.slug === slug);
+    let answer = doc && doc.fields.find(field => field.question.slug === slug);
```

Comment:
This is actually a `field`, not an `answer`. The client side model layer is structured like this:
```
document --(1:n)--> field --(1:1)--> question
                          \---(1:0)--> answer
```